site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,548 @@
+We'll use GitHub in this example as it's the most commonly used system for accessing code.
+
+To get started,
+you create a Read the Docs account,
+and link your GitHub account.
+Then you select the GitHub repository you'd like to build documentation for,
+at which point the magic happens.
+
+Read the Docs:
+
+* Clones your repository
+* Builds HTML, PDF, and ePub versions of your documentation from your ``master`` branch.
+* Indexes your documentation to allow full-text-search
+* Creates Version objects from each *tag* and *branch* in your repository, allowing you to optionally host those as well
+* Activates a webhook on your repository, so when you push code to any branch, your documentation is rebuilt
+
+Now,
+whenever you commit new code or documentation to your repository,
+your documentation is kept up to date.
+This works with your ``master`` branch,
+as well as any other branches or tags you might have activated documentation for.
+
+Read the Docs has two special versions:
+
+* ``latest`` which maps to the most up to date development version of your software
+* ``stable`` which maps to the latest tagged release of your software
+
+These are version aliases that are meant to maintain stable URL's for your users who want to use the most up to date committed or released software, respectively.
+
+We built Read the Docs to be "set it and forget it".
+Once you set your project up and activate the versions you want hosted,
+we sit downstream of your version control system and just keep your documentation up to date.
+It feels pretty magical once it's set up,
+and takes the thankless task of deploying documentation out of your day.
+
+Recommended Versioning System
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs only works with version control.
+This means that however you version your software,
+your documentation follows suite.
+This is great for integrating with your development team,
+but it also means you need to think about the proper strategy for versioning.
+
+If you work on a SaaS product,
+you probably only have one version.
+However,
+if you produce software with multiple supported versions,
+we recommend the following:
+
+* A ``master`` branch that the next release of your software is developed on
+* Git *branches* for ongoing maintenance of each version of your software that is maintained
+* Git *tags* for specific released versions that users might be using
+
+We recommend release branches because it allows you to update them over time.
+Git *tags* are static,
+so they are appropriate for specific versions that a user might have installed.
+
+An example:
+
+* ``master`` is your 2.2 release that isn't out yet
+* ``2.1.X`` is your release branch for the ``2.1`` version
+* ``2.1.0`` is a tag of your ``2.1`` branch, when you release ``2.1.0`` to your users
+* ``2.1.1`` is a similar tag of your ``2.1`` branch
+* ``1.4.X`` is the latest release branch of your ``1.X`` version
+* ``1.4.3`` is the latest release of your ``1.X`` version
+
```

Comment:
Not completely sure you need the previous section -- it's powerful stuff, but for an article at this level it might be too much detail? Although the general WTD Slack channel suggests it's something a lot of writers worry about, so it's a question only, not a recommendation.